web/tests/drafts.test.cjs

```diff
@@ -51,6 +52,36 @@ people.initialize_current_user(aaron.user_id);
 people.add_active_user(iago);
 people.add_active_user(zoe);
 
+const stream_A = {
+    subscribed: false,
+    name: "A",
+    stream_id: 1,
+};
+stream_data.add_sub(stream_A);
```

Comment:
I can merge this, since it's progress, but this should be using `web/tests/lib/example_stream.cjs` rather than adding an obviously incorrectly typed stream object.